pages/features/internationalization.mdx

```diff
@@ -0,0 +1,43 @@
+import { ImageTabs } from '../../components/ImageTabs';
+
+Start UI already implements internationalization, which means that it is designed to support multiple languages.
+It currently supports French, English, Arabic, and Swahili, and is set up to display text in these languages in a way that is appropriate
+for the language's writing system. For example, if you are using Start UI in Arabic, the text will be displayed right-to-left, as is standard for Arabic.
```

Comment:
Il manque la partie fontSize qui s'adapte.
Il faut aussi dire où se trouve la liste des langages et comment faire pour en ajouter un (créé une issue pour faire un guide sur l'ajout d'un nouveau langage, si ça existe pas déjà)

Reply:
- Do you have an example for the adaptative font size part ? 
- I've created the issue : https://github.com/BearStudio/start-ui-web-docs/issues
- I've added a link to the available locales

Reply:
Sorry, I didn't mention it before but it's not "the text will be displayed right-to-left" but "both the text and the UI will be ..."